README.md

```diff
@@ -43,6 +43,9 @@ sub init()
   ' Create the AblyTask
   m.ablyTask = createObject("roSGNode", "AblyTask")
 
+  ' Assign the API to the task used for authentication
+  m.ablyTask.key = "xVLyHw.XuwW-w:yOPxtXWlsZn10nzy"
```

Comment:
Where did this key come from? It _looks_ 'real' which is a bad idea. Can you replace with some kind of clearly redacted placeholder?

Reply:
Taken from a example page like this one: https://www.ably.io/documentation/core-features/authentication#basic-authentication

Mind you I don't remember witch one I grabbed it from.